index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Portfolio: Mobile Skeleton</title>
+        <link rel="stylesheet" href="index.css">
+    </head>
+
+    <body>
+        <main id="main-layout">
+            <header class="flex-item">
+                <nav class="nav-bar">
+                    <div class="nav-bar-item">
+                        <a href="#" title="logo">Welcome</a>
+                    </div>
+                    <div class="nav-bar-item">
+                        <img src="./static/images/hamburger-menu-icon.png" title="Menu Options" alt="Menu" width="30" height="30">
+                    </div>
+                </nav>
+            </header>
+
+            <div class="cont-preview flex-item">
+                <div>
```

Comment:
- [ ] Instead of using the `div` tag at line 23 as the container of your headline section, Kindly use the `section` tag. This will make your code look kinda professional and then you'll use one of the best practices for HTML.

Reply:
Fixed